src/Ed25519VerificationKey2018.js

```diff
@@ -428,6 +428,18 @@ function ed25519VerifierFactory(key) {
   };
 }
 
+function _isEqualBuffer(buf1, buf2) {
```

Comment:
@dlongley - since we're going to need this functionality in multiple repos, we should either extract it to a small standalone package, or see if there's an existing package we can use.

Also, since this is an isomorphic repo, will this Buffer-comparing code work in the browser, where it's a Uint8Array and not a buffer?

Reply:
Yes, it works with a Uint8Array.

Reply:
@dlongley sounds good. What are your thoughts on - does it make sense to implement it as a package? (just the `isEqualBuffer` function). Or is it small enough to just copy in each relevant repo?

Reply:
@dmitrizagidulin -- small enough to copy.